packages/tags/src/TagsEditor.tsx

```diff
@@ -47,31 +55,31 @@ const styles = {
   }),
 };
 
-const SortablePillHandle = SortableHandle(() => (
-  <div className={styles.handle}>
+const SortablePillHandle = SortableHandle((props: { isDisabled: boolean }) => (
+  <div className={`${styles.handle} ${props.isDisabled ? styles.pillDisabled : ''}`}>
```

Comment:
You can use the [cx](https://5faaafd0bd0f3f0008469537--emotion.netlify.app/docs/emotion#cx) helper here instead